src/injected/components/details-dialog.tsx

```diff
@@ -162,14 +167,30 @@ export class DetailsDialog extends React.Component<DetailsDialogProps, DetailsDi
     }
 
     private renderFileIssueButton(issueData: CreateIssueDetailsTextData): JSX.Element {
-        return (
+        const oldExperienceButton: JSX.Element = (
             <FileIssueDetailsButton
                 deps={this.props.deps}
                 issueDetailsData={issueData}
                 issueTrackerPath={this.state.issueTrackerPath}
                 restoreFocus={false}
             />
         );
+        const newExperienceButton: JSX.Element = this.state.userConfigurationStoreData ? (
+            <IssueFilingButton
+                deps={this.props.deps}
+                issueDetailsData={issueData}
+                userConfigurationStoreData={this.state.userConfigurationStoreData}
+                needsSettingsContentRenderer={IssueFilingNeedsSettingsHelpText}
+            />
+        ) : null;
```

Comment:
>) : null; [](start = 8, length = 9)

This feels inconsistent with how the other buttons work; for the inspect, back, and next buttons, the dialog handler exposes a "isButtonTypeDisabled" that owns deciding whether to disable a button and the buttons themselves are passed a disabled prop so they can decide how to render a "disabled" vs "enabled" state, but here, the dialog's renderer takes on both of those 2 responsibilities.

Do the other buttons treat "disabled" as meaning "don't render at all" or "render as disabled"? I think it would be good to be consistent in both code layout and UX.

Reply:
Yeah, I also think that we should be consistent. We have created an issue (https://github.com/Microsoft/accessibility-insights-web/issues/547) to change the other buttons' UX to remove their disabled state.

Reply:
Updated!